airbyte-integrations/connectors/source-kyve/source_kyve/schemas/bitcoin/block.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "type" : "object",
+  "properties" : {
+    "hash" : {
+      "type" : "string"
+    },
+    "height" : {
```

Comment:
For each schema, can you add "AdditionalProperties": True?

Reply:
Resolved